examples/02_Parameter Estimation.ipynb

```diff
@@ -993,31 +997,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This is even better. Now we have an \"ok\" estimate, ~150 mV (for the sake of a demo). The estimate could be refined further by setting a lower tolerance (tol parameter), or repeating the 4 parameter estimation steps, above."
+    "This is even better. Now we have an \"ok\" estimate, ~150 mV (for the sake of a demo). The estimate could be refined further by setting a lower tolerance (tol parameter), or repeating the 4 parameter estimation steps, as shown above.\n",
+    "\n",
+    "Parameter estimation is also limited by the model itself. This is a simplified battery model, meaning there were some simplifying assumptions made. It will likely not be able to capture the behavior of a model as well as a higher fidelity model (e.g., BatteryElectroChemEOD)."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Parameter estimation is also limited by the model itself. This is a simplified battery model, meaning there were some simplifying assumptions made. It will likely not be able to capture the behavior of a model as well as a higher fidelity model (e.g., BatteryElectroChemEOD)."
+    "## Conclusion"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Closing\n",
+    "This chapter introduced the concept of parameter estimation, through which the parameters of a physics-based model are estimated. This is done using a mixture of data, knowledge (e.g., from system specs), and intuition. For large, complex models, it can be VERY difficult and computationally expensive. Fortunately, in this case we have a relatively simple model.\n",
     "\n",
-    "This chapter introduced the concept of parameter estimation, through which the parameters of a physics-based model are estimated. This is done using a mixture of data, knowledge (e.g., from system specs), and intuition. For large, complex models, it can be VERY difficult and computationall expensive. Fortunately, in this case we have a relatively simple model.\n",
+    "In ProgPy a models `estimate_params` method is used to estimate the parameters. See the [parameter estimation documentation](https://nasa.github.io/progpy/prog_models_guide.html#parameter-estimation) for more details.\n",
     "\n",
-    "In ProgPy a models `estimate_params` method is used to estimate the parameters. See [Parameter Estimation Docs](https://nasa.github.io/progpy/prog_models_guide.html#parameter-estimation) for more details."
+    "In the next notebook, we will be exploring (see __[03 Existing Models](03_Existing%20Models.ipynb)__)."
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.12.0 64-bit",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
Are the changes on this line, line 1040, and line 1049 supposed to be checked in, or a result of running it locally on your machine?

Reply:
Updated - metadata changes just a result of the kernel I ran it on